cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/internal/SSAConstruction.qll

```diff
@@ -1319,6 +1319,13 @@ predicate canReuseSsaForMemoryResult(Instruction instruction) {
   // We don't support reusing SSA for any location that could create a `Chi` instruction.
 }
 
+predicate hasIncompleteSsa(IRFunction f) {
```

Comment:
I understand this may be more or less an implementation detail, but should we still put a doc comment here?

Reply:
Happy to do that. Note that this is still a draft since I wanted to check the implications on DCA before I made this more pretty 🙂

Reply:
I've force-pushed some QLDoc in 510df38da2476708271777dc17ad5bd52068f14c